polaris/polaris/deposit/views.py

```diff
@@ -191,11 +144,6 @@ def deposit(account: str, request: Request) -> Response:
     except Ed25519PublicKeyInvalidError:
         return render_error_response("invalid 'account'")
 
-    # Verify the optional request arguments.
-    verify_optional_args = _verify_optional_args(request)
-    if verify_optional_args:
-        return verify_optional_args
-
```

Comment:
We're not doing anything with these optional args, `memo` and `memo_type`, so I removed the check. If we want to support the optional args we should file an issue and add full support for optional arguments on all endpoints as a separate effort.